fenix/app/src/main/res/values/static_strings.xml

```diff
@@ -127,4 +127,21 @@
     <string name="translations_bottom_sheet_translating_in_progress">In progress</string>
     <string name="translations_bottom_sheet_translating_in_progress_content_description">Translating in Progress</string>
 
+
+    <string name="translation_option_bottom_sheet_title">Translation Options</string>
+    <string name="translation_option_bottom_sheet_always_translate">Always offer to translate</string>
+    <string name="translation_option_bottom_sheet_always_translate_in_language">Always translate %1$s</string>
+    <string name="translation_option_bottom_sheet_never_translate_in_language">Never translate %1$s</string>
+    <string name="translation_option_bottom_sheet_never_translate_site">Never translate this site</string>
+    <string name="translation_option_bottom_sheet_translation_settings">Translation settings</string>
+    <string name="translation_option_bottom_sheet_about_translations">About translations in %1$s</string>
+
+    <string name="translation_settings_fragment_toolbar_title">Translations</string>
```

Comment:
```suggestion
    <string name="translation_settings_toolbar_title">Translations</string>
```
Let's not attach the word "fragment" into our string IDs since it's used in a Composable.

Reply:
I think I need to delete and bottom_sheet because it's the same.

Reply:
Done